internal/terraform/node_action_trigger_instance_plan.go

```diff
@@ -120,9 +121,23 @@ func (n *nodeActionTriggerPlanInstance) Execute(ctx EvalContext, operation walkO
 		return diags
 	}
 
+	// We remove the marks for planning, we will record the sensitive values in the plans.ActionInvocationInstance
+	unmarkedConfig, pvms := actionInstance.ConfigValue.UnmarkDeepWithPaths()
+	// We only support sensitive marks, all other marks cause an error
```

Comment:
Is there another ticket / TODO for ephemeral values? Or are we explicitly not supporting them?